kolibri/plugins/learn/assets/src/views/index.vue

```diff
@@ -39,26 +39,28 @@
 
     <div v-if="!isSearchPage">
       <tabs>
-        <tab-link
-          type="icon-and-title"
-          :title="$tr('recommended')"
-          icon="forum"
-          :link="recommendedLink"
-        />
-        <tab-link
-          type="icon-and-title"
-          :title="$tr('topics')"
-          icon="folder"
-          :link="topicsLink"
-        />
-        <tab-link
-          name="exam-link"
-          v-if="isUserLoggedIn && userHasMemberships"
-          type="icon-and-title"
-          :title="$tr('exams')"
-          icon="assignments"
-          :link="examsLink"
-        />
+        <template scope="tab">
+          <component
+            :is="tab.link"
```

Comment:
I agree. I think the HTML argument made above is really good, so we can leave it as is. I'm curious about why we're not just passing in an array or something, though. I understand that this is more explicit, it just seems like we're writing a bunch of repetitious code for no particular reason.[ I liked the way VueJS did it in their example](https://vuejs.org/v2/guide/components.html#Scoped-Slots):
```
<my-awesome-list :items="items">
  <!-- scoped slot can be named too -->
  <template slot="item" scope="props">
    <li class="my-fancy-item">{{ props.text }}</li>
  </template>
</my-awesome-list>
```